src/main/java/io/stargate/sgv2/jsonapi/service/processor/HybridFieldExpander.java

```diff
@@ -76,9 +104,15 @@ private static void addFromObject(ObjectNode doc, ObjectNode hybrid, int docInde
     lexical =
         validateSubFieldType(
             lexical, DocumentConstants.Fields.LEXICAL_CONTENT_FIELD, docIndex, docCount);
+    if (!lexical.isNull()) {
```

Comment:
this and the vectorize below should be moved to be tracked in the insertOne and insertMany commands. 

the only thing this needs to do is track if the user include $hybrid in the command.